apiary-metastore-events/apiary-hive-events/src/main/java/com/expediagroup/apiary/extensions/events/metastore/event/ApiaryInsertEvent.java

```diff
@@ -34,13 +37,26 @@
 
   public ApiaryInsertEvent(InsertEvent event) {
     super(event);
-    databaseName = event.getDb();
-    tableName = event.getTable();
-    partitionKeyValues = event.getPartitionKeyValues();
+
+    Table table = event.getTableObj();
+    databaseName = table.getDbName();
+    tableName = table.getTableName();
+
+    partitionKeyValues =  new LinkedHashMap<>();
```

Comment:
Just checking - in the previous implementation it also returned an empty map rather than null if there were none?

Reply:
Yup! This snippet is actually ported directly from the old codebase: https://github.com/apache/hive/blob/rel/release-2.3.7/metastore/src/java/org/apache/hadoop/hive/metastore/events/InsertEvent.java#L63-L71

Reply:
OK, cool.